nodes/nomos-node-api/tests/todo.rs

```diff
@@ -71,7 +74,7 @@ impl Backend for WebServer {
         Ok(Self { addr: settings })
     }
 
-    async fn serve(self) -> Result<(), Self::Error> {
+    async fn serve(self, _handle: OverwatchHandle) -> Result<(), Self::Error> {
```

Comment:
Maybe I'm missing something from previous PRs, but may I ask why this handle is not used?

Reply:
Indeed, I think it is because is feed to the settings. @al8n ? I think in this case it should not be part of the settings itself. But should be expected to be received later on just before beeing served by the Backend. Unless there is a strong reason we can not.

Reply:
> Maybe I'm missing something from previous PRs, but may I ask why this handle is not used?

This handle is used to get a `Relay` for different services, in this test case, we only have one service (self), so this handle is not used.

Reply:
Ah, I"m sorry. I didn't realize that this is a test :)